src/zip.ts

```diff
@@ -36,7 +36,7 @@ export function contentLength(files: Iterable<Metadata>) {
   return centralLength + offset
 }
 
-export async function* loadFiles(files: ForAwaitable<ZipEntryDescription & Metadata>) {
+export async function* loadFiles(files: ForAwaitable<ZipEntryDescription & Metadata>, markerBeforeFileStart: any = null, markerAfterFileEnd: any = null) {
```

Comment:
The second argument should be an Options object, otherwise this will become arguments hell when other options come in.